### PR TITLE
bundles/hg: add support for Mercurial SCM (aka "hg").

### PR DESCRIPTION
**Pull request by mfxmfx**
Copied and adapted from bundles/git.

### CODE REVIEW

File: bundles/hg/hg.moon [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+class Hg
+  new: (root, hg_dir) =>
+    @root = root
+    @hg_dir = hg_dir
+    @name = 'Hg'
+
+  -- returns a list of hg files, including untracked
+  files: =>
+    --output = @run 'manifest'
```
Comment (kirbyfan64) [inline bundles/hg/hg.moon]: Leftovers?

Comment (mfxmfx) [inline bundles/hg/hg.moon]: Documentation.

**Comment by mfxmfx** [comment]
Rebased on current master and removed comments as requested by @kirbyfan64 .

**Comment by mfxmfx** [comment]
Please merge or reject so that I can clean up my "quilt" patches. Thanks.

File: .travis.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ addons:
   apt:
     packages:
       - libgtk-3-dev
+      - mercurial
```
Comment (shalabhc) [inline .travis.yml]: Why is this needed (given git isn't)?

**Comment by shalabhc** [comment]
I tried this out and seems to work great. @nilnor - can we merge (once travis.yml is resolved)?

**Comment by mfxmfx** [comment]
Updated - I had not been aware that mercurial (which is need for the test-suite in ```bundles/hg/spec/hg_spec.moon```) is pre-installed on Travis anyway.

**Comment by nilnor** [comment]
Looks great, thanks!
